pennylane/ops/qubit.py

```diff
@@ -1822,9 +1861,69 @@ def adjoint(self):
         return IsingXX(-phi, wires=self.wires)
 
 
+class IsingYY(Operation):
+    r"""IsingYY(phi, wires)
+    The Ising YY coupling gate
+
+    .. math:: \mathtt{YY}(\phi) = \begin{bmatrix}
+        \cos(\phi / 2) & 0 & 0 & i \sin(\phi / 2) \\
+        0 & \cos(\phi / 2) & -i \sin(\phi / 2) & 0 \\
+        0 & -i \sin(\phi / 2) & \cos(\phi / 2) & 0 \\
+        i \sin(\phi / 2) & 0 & 0 & \cos(\phi / 2)
+        \end{bmatrix}.
+
+    **Details:**
+
+    * Number of wires: 2
+    * Number of parameters: 1
+    * Gradient recipe:
+
+    .. math::
+         \frac{d}{d\phi} f(YY(\phi))
+         = \frac{1}{2} \left[ f(YY(\phi + \pi/2)) - f(YY(\phi - \pi/2)) \right]
+
+     where :math:`f` is an expectation value depending on :math:`YY(\phi)`.
```

Comment:
The [rendering here seems to be a bit off](https://pennylane--1358.org.readthedocs.build/en/1358/code/api/pennylane.IsingYY.html?highlight=isingyy#pennylane.IsingYY), hopefully, the extra spaces will help:
![Screenshot from 2021-06-14 13-18-35](https://user-images.githubusercontent.com/24476053/121932670-21cba900-cd13-11eb-8bd9-b63bca3d78c0.png)

The output after each change can be checked locally by running `make docs` or by having a look at the documentation continuous integration check at the bottom of the Pull Request. Clicking on the "Details" button will link to a changed version of the docs:

 ![Screenshot from 2021-06-14 13-21-02](https://user-images.githubusercontent.com/24476053/121933017-84bd4000-cd13-11eb-8e4d-f921348ca073.png)


There, it can be checked that this operation appears well e.g., by searching for `IsingYY`.
```suggestion
       where :math:`f` is an expectation value depending on :math:`YY(\phi)`.
```